static/app/views/issueDetails/streamline/solutionsSection.tsx

```diff
@@ -140,7 +149,10 @@ export default function SolutionsSection({
           </ExpandButton>
         </ResourcesWrapper>
       )}
-      {(hasSummary || aiNeedsSetup) && (
+      {(aiNeedsSetup ||
+        hasAutofix ||
+        autofixNeedsSetup ||
```

Comment:
I may be misreading your code, but are you accounting for issues that shouldn't have AI/Autofix at all when saying that this button should show if `aiNeedsSetup` or `autofixNeedsSetup`? We don't want to show this button on those issues even if AI/Autofix are not set up.